packages/pf-codemods/index.js

```diff
@@ -13,6 +13,7 @@ program
   .option('--exclude <rules>', 'Run recommended rules EXCLUDING this comma-seperated list')
   .option('--fix', 'Whether to run fixer')
   .option('--format <format>', 'What eslint report format to use', 'stylish')
+  .option('--no-cache', 'Disables eslint caching')
```

Comment:
This + line 76 add a CLI option that allows us to selectively use eslint caching. Defaults to true so that consumers using it still get the benefits of the caching.

You can read more about what I'm actually doing here if you're curious: https://www.npmjs.com/package/commander#options